code/style.css

```diff
@@ -1,4 +1,212 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*GOOGLE FONT*/
+@import url('https://fonts.googleapis.com/css2?family=Staatliches&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap');
+
+.container {
+    width: 320px;
+    margin: 25px auto;
 }
+
+/**** HEADER ****/
+.header-landing-page {
+    background: #13141b;
+    height: 100px;
+    display:flex;
+    align-items: center;
+    padding: 0 16px;
+    margin: -8px;
+    font-family: 'Staatliches', cursive;
+    justify-content: space-between;    
+}
+
+.header-logo {
+    width: 180px;
+}
+.header-logo-container {
+    overflow:hidden;
+    width: 100px;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: center;   
+}
+
+.foot-step-right-side {
+    margin:5px;
+    display: none;
+}
+
+.site-title {
+    color:white;
+    font-size: 58px;
+    display: flex;
+}
+.navigation-container {
+    display: none;
+    font-size: 24px;    
+    background-color: #13141b;
+    color:rgb(9, 85, 32);     
```

Comment:
if possible, stick to one type of color. Either rgb or hex. This makes the code look tidier